libraries/AP_Airspeed/AP_Airspeed_Backend.cpp

```diff
@@ -28,6 +28,7 @@ AP_Airspeed_Backend::AP_Airspeed_Backend(AP_Airspeed &_frontend, uint8_t _instan
     frontend(_frontend),
     instance(_instance)
 {
+    frontend.state[instance].use_zero_offset = frontend.param[instance].options & (uint8_t)Option::USE_ZERO_OFFSET;
```

Comment:
Needs a !=0 to make it boolean, otherwise the result here is 2.

Reply:
Fixed, thanks!